src/middleware/auth/logto.ts

```diff
@@ -49,83 +49,96 @@ export class LogToHelper {
     return this.allResourceWithNames
   }
 
-  public async setDefaultRoleForUser(userId: string): Promise<ILogToErrorResponse | void> {
+  public async setDefaultRoleForUser(userId: string): Promise<ILogToErrorResponse> {
     const roles = await this.getRolesForUser(userId)
-    if (roles && roles.status === 200) {
+    if (roles.status === 200) {
```

Comment:
Would cross-check with @abdulla-ashurov since he worked on this, but is the status code still 200 when a user has no roles? (This is the current logic as far as I follow, checks that the response actually has roles in it, not just whether status code is 200)

Reply:
Status == 200 here means that `this.getRolesForUser(userId)`  method finished successful. I made such internal structure on purpose cause here we could pass the same structure as a response to user with the same code and use the `error` field as the reason of error.
Also, for the first bootstrap thing we need to check is there any roles assigned to the user as a sanity check and if it's empty - assign TestNet role.